sound/soc/codecs/rt1308-sdw.c

```diff
@@ -132,9 +132,9 @@ static int rt1308_read_prop(struct sdw_slave *slave)
 	/* for sink */
 	nval = hweight32(prop->sink_ports);
 	num_of_ports += nval;
-	prop->sink_dpn_prop = devm_kcalloc(&slave->dev, nval,
-						sizeof(*prop->sink_dpn_prop),
-						GFP_KERNEL);
+	prop->sink_dpn_prop = devm_kzalloc(&slave->dev,
+					   nval * sizeof(*prop->sink_dpn_prop),
+					   GFP_KERNEL);
```

Comment:
is it the same patch as in a separate PR? In any case this is redundant.

Reply:
kcalloc does not reset the memory to zero in my tests.

Reply:
removed